tests/smoke/base.sh

```diff
@@ -55,6 +55,9 @@ elif command -v maturin &> /dev/null; then
   pip install -e .
 elif [ "$(pip list|grep -c scikit_build_core)" -ne "0" ]; then
   pip install -e .
+elif [ "$(pip list|grep -c pybind11)" -ne "0" ]; then
+  # Assuming you are inside the root of the CMake source directory
+  pip install .   
```

Comment:
This should still be `-e`

Reply:
in my last PR I removed `-e` from everywhere, so we will be sure to test the package correctly (as much as possible). my main concern is about testing this extension files .. so we would ensure to test if that is in the package .. maybe the test is not good enough yet ... but at least it is a first step